docs/support.mdx

```diff
@@ -1,111 +1,35 @@
 ---
-title: Getting help
+title: Support
 description: How to find answers to questions or problems you encounter while using IBM Quantum (Platform, or on IBM Cloud) or Qiskit Runtime
 
 ---
 
-# Getting help
+# Support
 
-## IBM Quantum Support
+## How to get help
 
-Members of the IBM Quantum&trade; Network can reach out to IBM Quantum Support if they have technical difficulties, questions, or concerns. Ask your administrator or IBM&reg; representative for IBM Quantum Support contact information.
+| | |
+| --- | --- |
+| IBM Quantum&trade; Network members  | Reach out to IBM Quantum Support if you have technical difficulties, questions, or concerns. Ask your administrator or IBM&reg; representative for IBM Quantum Support contact information. |
+| Pay-As-You-Go users  | Open a case in the IBM Cloud&reg; Support Center by following [this link](https://cloud.ibm.com/unifiedsupport/cases/form), and choosing 'Qiskit Runtime' from the Topic field. |
```

Comment:
I'm not a fan of the link text being "this link".  Can we change it to "by filling out [this form] or something?